pkg/create/templates/scripts.go

```diff
@@ -51,7 +51,7 @@ To use it, install S2I: https://github.com/openshift/source-to-image
 
 Sample invocation:
 
-s2i build git://<source code> {{.ImageName}} <application image>
+s2i build <source code path/URL> {{.ImageName}} <application image>
```

Comment:
i'm with @rhcarvalho, s2i is probably more commonly used with "." as the path, anyway i don't think the ordering is critical.